.github/workflows/legal-api-cd.yml

```diff
@@ -1,11 +1,6 @@
 name: Legal API CD
 
 on:
-  push:
-    branches:
-      - main
-    paths:
-      - "legal-api/**"
```

Comment:
Disable auto-push from main branch.

This means a developer will have to manually run this CD to deploy release branch to Test. This is a good idea anyway so we sync with Riyaz before deploying.